momo_modules/shared/pic12/src/peripheral/timer1.h

```diff
@@ -34,8 +34,11 @@ enum
 	TMR1L = value & 0xFF; \
 	TMR1H = (value >> 8);
 
-void tmr1_config(uint8_t config);
-//void tmr1_load(uint8_t high, uint8_t low);
-void tmr1_setstate(PeripheralState state);
+#define tmr1_config(config) T1CON = config
+
+#define tmr1_setstate(state) \
```

Comment:
This should be wrapped in { } brackets so that if you put it after an if statement you don't get surprised when only the first line is conditional.